packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -300,20 +285,40 @@ describe.only("Sequence.Revertibles with Remote Edits", () => {
 		const id = collection.add(0, 5, IntervalType.Simple).getIntervalId()!;
 		containerRuntimeFactory.processAllMessages();
 
+		collection2.change(id, 3, 8);
+		containerRuntimeFactory.processAllMessages();
```

Comment:
I don't think this and the next test are necessary. I don't think there's anything conflicting between the two clients because client 2's change is seen by both clients before client 1 does a revertible edit + revert. So it's the same as revertible delete + revert over any baseline (in this case, there's agreement among clients that the interval is at [3,8) before any revertible changes) which is mostly covered by the local tests above. You can either get rid of the processAllMessages after the client 2 edit so client 1 doesn't see it until after its local delete, or turn these into a single client test with ack (if that case isn't already covered above).